@ndlib/gatsby-theme-marble/src/components/Common/form.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import InputText from '../Common/inputText'
+import InputTextRadio from '../Common/inputRadio'
+import Recaptcha from 'react-google-invisible-recaptcha'
+
+import style from './style.module.css'
+import InputBox from './inputBox';
+
+class Form extends Component {
```

Comment:
This should be rewritten as a functional component to match the design patterns of the rest of the project.
```
const Form = () => { ...
```